src/actions/channels.ts

```diff
@@ -1006,7 +1008,7 @@ export function searchAllChannels(term: string, notAssociatedToGroup = '', exclu
             },
         ]), getState);
 
-        return {data: channels};
+        return {data: response};
```

Comment:
Why do we use `response` here instead of `channels`?

Reply:
It is still an array of channels for non-paginated searches, however for paginated searches it is returned as an object like this:
```js
{
  channels: [],
  total_count: 0
}
```

Reply:
Yes, I guess my question was more about the reason behind breaking consistency and returning two different structures. Isn't `total_count` equal to `channels.length`?

Reply:
`total_count` is the total number of channels that match the query regardless of what page of results are being requested and regardless of the number of results per page.

In terms of your consistency question we have two options that I can see:

1. Make a new action so that there is one action for paginated queries and another for non-paginated queries. 
2. Have one action that returns a different structure depending on whether the query is requested paginated or not.

I don't really have a preference so I went with the one that _seemed_ to have less code to write (option 2). That said I'm totally open to option 1.

Reply:
That makes sense. Thank you for clarifying @mkraft :+1: